internal/admin/dashboard/templates/page-models.html

```
@@ -0,0 +1,341 @@
+    <div class="alert alert-warning" x-show="!aliasesAvailable && !authError">
+        Aliases feature is unavailable.
+    </div>
+    <div class="alert alert-warning" x-show="aliasError && !authError" x-text="aliasError"></div>
+    <div class="alert alert-success" x-show="aliasNotice" x-text="aliasNotice"></div>
+    <div class="alert alert-warning" x-show="modelOverrideError && !authError && !modelOverrideFormOpen" x-text="modelOverrideError"></div>
+    <div class="alert alert-success" x-show="modelOverrideNotice && !modelOverrideError" x-text="modelOverrideNotice"></div>
+
+    <!-- Category Tabs -->
+    <div class="category-tabs" x-show="categories.length > 0">
+        <template x-for="cat in categories" :key="cat.category">
+            <button class="category-tab" :class="{ active: activeCategory === cat.category }" @click="selectCategory(cat.category)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)